docker-compose.yml

```diff
@@ -39,6 +39,7 @@ services:
       - MYINFO_CLIENT_ID=mockClientId
       - MYINFO_CLIENT_SECRET=mockClientSecret
       - WEBHOOK_SQS_URL=http://localhost:4566/000000000000/local-webhooks-sqs-main
+      - INTRANET_IP_LIST_PATH=tests/mock-intranet-ips.txt
```

Comment:
does this need to be added to docker?

Reply:
? this is the docker-compose file

Reply:
as discussed, see https://github.com/opengovsg/FormSG/pull/2131/commits/849a406d944d06ff91b945542351eaddbba31d30